docs/style/entities/work.md

```diff
@@ -0,0 +1,85 @@
+# Work
+
+A work is a distinct, notable literary creation. The work entity captures
+information about the *content* that was created, not any physical or digital
+representation.
+
+## Distinctiveness
+
+A work is considered distinct based on the substantive uniqueness of its
+content. A work is distinct from another if it is revised, adapted, translated
+or reconstructed. Content is not considered a distinct work if it only
+contains minor variations such as spelling or typographical differences.
+
+Sometimes content is integral to a work and is not considered distinct from
+it. In general, chapters of a novel are *not* considered to be their own
+distinct works. Poems, essays, articles and short stories *are* generally
+considered distinct works.
+
+## Naming
+
+The name of a work should be its canonical title, in the language it was
+written by its author, or translator of translated works. Additional names,
+such as descriptive titles, known variations or transliterations should be
+included as aliases.
+
+The language of a work’s name can differ from the language of the actual work.
+For example, a work written in English can have a Latin name. The name of the
+work should be capitalized based on the rules of the language of its name.
+
+## Untitled works
+
+If a work does not have a title, the name of the work should be the first line
+or sentence of the work’s text, enclosed in square brackets. Do not apply
+capitalization rules; retain the case of the original text. Examples:
+
+- [At that hour when all things have repose]
+- [Rash mortal! stay thy feet and look around]
```

Comment:
We could possibly add a link to the entities on BB like you do with the special case authors below, what do you think?
Perhaps a link next to the title? 

https://bookbrainz.org/work/c0b50c4b-d57d-43bf-83b2-4930d938a0e4
https://bookbrainz.org/work/ed6d8c28-fe3c-4fde-b399-6663c2225805

Reply:
Added.